src/lib/cat_superhero.svelte

```diff
@@ -4,46 +4,107 @@
 -->
 <script lang="ts">
   import { base } from '$app/paths';
-
-  type RowData = {
-    title: string;
-    text: string;
-  };
-
+  import type { RowData } from './types';
+  import AccordionItem from './accordion_item.svelte';
   const rows: RowData[] = [
     {
       title: 'Find',
-      text: 'Find a cat who needs your help'
+      text: 'Find a homeless cat who needs your help',
```

Comment:
```suggestion
      text: 'Find a homeless cat who needs your help.',
```
To match how it was displayed on the Figma.